server/datastore/mysql/invites.go

```diff
@@ -182,3 +185,37 @@ func (d *Datastore) loadTeamsForInvites(ctx context.Context, invites []*fleet.In
 
 	return nil
 }
+
+func (d *Datastore) UpdateInvite(ctx context.Context, id uint, i *fleet.Invite) (*fleet.Invite, error) {
+	return i, d.withRetryTxx(ctx, func(tx sqlx.ExtContext) error {
+		query, args, err := dialect.Update("invites").Where(goqu.Ex{"id": id}).Set(i).ToSQL()
```

Comment:
Hmm where did that "we'll only use it when needed" promise go? :D Just kidding though, that doesn't look too bad once you know how `Set` works.

Reply:
Hah, yeah, I thought I would give it a try. Happy to go back to the old ways if that's what feels better.